File: TodoApi/Models/CreatePatientRequest.cs
public class CreatePatientRequest
{
    public string? Username {get; set;}
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string Email { get; set; }
    public string Birthday { get; set; } // Formato: DD/MM/YYYY
    public string Gender { get; set; } 
    public int MedicalNumber { get; set; }
    public string Phone { get; set; }
        public string EmergencyContact { get; set; }

    public List<string> MedicalConditions { get; set; } = new List<String> (); // Condições médicas
}

File: TodoApi/healthcare-app/src/app/Models/CreatePatientRequest.ts
// src/app/models/create-patient-request.ts
export interface CreatePatientRequest {
    username?: string;
    firstName: string;
    lastName: string;
    email: string;
    birthday: string; // Format: DD/MM/YYYY
    gender: string;
    medicalNumber: number;
    phone: string;
    emergencyContact: string;
    medicalConditions: string[];
  }
  